pkg/minishift/constants/constants.go

```diff
@@ -52,7 +52,10 @@ func ProfilePrivateKeyPath() string {
 	return filepath.Join(constants.Minipath, "certs", "id_rsa")
 }
 
-func GetOpenshiftImageName(openshiftVersion string) string {
+func GetOpenshiftImageName(openshiftVersion string, isGreaterOrEqualToBaseVersion bool) string {
```

Comment:
I had to change the function signature if I try to import `openshiftVersion "github.com/minishift/minishift/pkg/minishift/openshift/version"` and consume `openshiftVersion.IsGreaterOrEqualToBaseVersion(openShiftVersion, constants.RefactoredOcVersion)` then it become a cyclic dependency and build fail.

Reply:
is this ONLY for the `oc` command or general usage? I am confused about the function name in case of this change...

since we only use this image to copy `oc` out or is it more?

Reply:
@gbraad yes this is only for copy oc command nothing more.

Reply:
in that case it is better to change the funcgtion name to reflect this better

Reply:
or if this function is used in more place, it is better ot break this out and specify that a choice is needed for the pull of the container for the `oc` command

Reply:
@gbraad It only used one place, does `GetOpenshiftCliImage` sound better?

Reply:
makes more sense